test/zdtm.py

```diff
@@ -1997,7 +1997,8 @@ def run_test(self, name, desc, flavor):
             raise Exception("The kernel is tainted: %r (%r)" %
                             (taint, self.__taint))
 
-        if test_flag(desc, 'excl'):
+        link_remap_excl = '--link-remap' in desc.get('opts', '').split() + desc.get('dopts', '').split()
```

Comment:
pls write the comment why they are exclusive

Reply:
done.